conans/client/deps_builder.py

```diff
@@ -269,20 +280,17 @@ def get_graph_updates_info(self, deps_graph):
         return {conan_reference: self._retriever.update_available(conan_reference)
                 for conan_reference, _ in deps_graph.nodes}
 
-    def load(self, conan_ref, conanfile):
-        """ compute the dependencies graph for:
-        param conan_ref: ConanFileReference for installed conanfile or path to user one
-                         might be None for user conanfile.py or .txt
-        """
+    def load(self, conanfile):
+
         dep_graph = DepsGraph()
         # compute the conanfile entry point for this dependency graph
-        root_node = Node(conan_ref, conanfile)
+        root_node = Node(None, conanfile)
```

Comment:
Yes, that is right, I have reviewed its usage, and it seems all collaborators using the consumer name, like "PROJECT" are receiving it as parameter.